src/Microsoft.DotNet.Wpf/src/WindowsBase/WindowsBase.csproj

```diff
@@ -439,6 +439,14 @@
     </ProjectReference>
   </ItemGroup>
   
+  <ItemGroup>
+     <!--
+     Allow test projects access to internals (but only those that require it). Arcade will generate a file like
+     "WindowsBase.InternalsVisibleTo.cs" with the proper assembly name and PublicKey for us
+     -->
+     <InternalsVisibleTo Include="DrtPackagingApis" />
```

Comment:
In our codebase, we use `ExtendedAssemblyInfo` for this. See example [here](https://github.com/dotnet/wpf/blob/031b79d9aec73dc35d17792f1446a60423db564a/eng/WpfArcadeSdk/tools/ExtendedAssemblyInfo.props#L10)